recipes/libpng/all/conanfile.py

```diff
@@ -26,6 +27,7 @@ class LibpngConan(ConanFile):
         "sse": [True, False],
         "vsx": [True, False],
         "api_prefix": ["ANY"],
+        "with_zlibng": [True, False],
```

Comment:
```suggestion
        "with_zlib": ["zlib", "zlibng"],
```
Maybe future-proof it a bit.